lib/pbench/server/database/models/server_settings.py

```diff
@@ -54,32 +55,34 @@ def __str__(self) -> str:
         return f"Missing key value in {self.name!r}: {self.cause}"
 
 
-class ServerConfigMissingKey(ServerConfigError):
-    """Attempt to set a ServerConfig with a missing key."""
+class ServerSettingMissingKey(ServerSettingError):
+    """Attempt to set a ServerSetting with a missing key."""
 
     def __str__(self) -> str:
-        return "Missing key name"
+        return "Missing server setting key name"
 
 
-class ServerConfigBadKey(ServerConfigError):
-    """Attempt to commit a ServerConfig with a bad key."""
+class ServerSettingBadKey(ServerSettingError):
+    """Attempt to commit a ServerSetting with a bad key."""
 
     def __init__(self, key: str):
         self.key = key
 
     def __str__(self) -> str:
-        return f"Configuration key {self.key!r} is unknown"
+        return f"Server setting key {self.key!r} is unknown"
 
 
-class ServerConfigBadValue(ServerConfigError):
-    """Attempt to assign a bad value to a server configuration option."""
+class ServerSettingBadValue(ServerSettingError):
+    """Attempt to assign a bad value to a server setting option."""
 
     def __init__(self, key: str, value: JSONVALUE):
         self.key = key
         self.value = value
 
     def __str__(self) -> str:
-        return f"Unsupported value for configuration key {self.key!r} ({self.value!r})"
+        return (
+            f"Unsupported value for server settings key {self.key!r} ({self.value!r})"
```

Comment:
Should `settings` be plural here?  (I think there's only one in play....)

Reply:
Agreed.

Reply:
Fixed.